processor/transaction/package_tests/TestProcessTransactionFull.approved.json

```diff
@@ -119,6 +119,10 @@
                 },
                 "id": "945254c5-67a5-417e-8a4e-aa29efcbfb79",
                 "marks": {
+                    "another_mark": {
```

Comment:
need to run approvals again, these are now called `some_long` `some_float`

Reply:
thanks, done